src/examples/napi_hello/napi_hello.cpp

```diff
@@ -65,10 +73,12 @@ int main(int argc, char* argv[]) {
     std::shared_ptr<Region> encoder   = net.addRegion("encoder",   "RDSEEncoderRegion", encoder_params);
     std::shared_ptr<Region> sp_global = net.addRegion("sp_global", "SPRegion",   sp_global_params);
     std::shared_ptr<Region> tm        = net.addRegion("tm",        "TMRegion",   tm_params);
+    std::shared_ptr<Region> output 	  = net.addRegion("output",    "FileOutputRegion", "{outputFile: '"+ output_file + "'}");
 
     // Setup data flows between regions
     net.link("encoder",   "sp_global", "", "", "encoded", "bottomUpIn");
     net.link("sp_global", "tm",        "", "", "bottomUpOut", "bottomUpIn");
+    net.link("tm", "output", "UniformLink", "", "bottomUpOut", "dataIn");
```

Comment:
nit: "UniformLink", or anything in this argument, is ignored.  This is a hold-over from the NuPIC code.